include/grpc/impl/codegen/grpc_types.h

```diff
@@ -213,13 +213,15 @@ typedef struct {
 #define GRPC_ARG_SERVICE_CONFIG "grpc.service_config"
 /** LB policy name. */
 #define GRPC_ARG_LB_POLICY_NAME "grpc.lb_policy_name"
-/** Server name. Not intended for external use. */
```

Comment:
Suggest moving to an internal header and out from the public interface if it's not intended for external use.